config/packages/security.yaml

```diff
@@ -85,6 +85,9 @@ sulu_security:
         enabled: true
     password_policy:
         enabled: true
+    two_factor:
+        force:
```

Comment:
it can be used as a boolean to activate it for all via:

```yaml
security:
   two_factor:
       force: true
```

But to limit it to only some email adresses it can be done via:


```yaml
security:
   two_factor:
       force:
           pattern: '/@sulu\.io$/'
```

Reply:
keep `force`